tldw_Server_API/app/api/v1/endpoints/audio/audio_transcriptions.py

```
@@ -453,11 +473,45 @@ async def create_transcription(
     seg_embeddings_provider: Optional[str] = Form(default=None, description="Embeddings provider override for TreeSeg"),
     seg_embeddings_model: Optional[str] = Form(default=None, description="Embeddings model override for TreeSeg"),
     current_user: User = Depends(get_request_user),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)